src/components/Checkbox/checkbox.css

```diff
@@ -16,14 +16,19 @@ input {
 }
 
 .form-check {
-  display: block;
+  display: flex;
   margin-bottom: var(--sgds-spacer-2);
   min-height: 2rem;
+  flex-direction: row;
 }
 
 .form-check .form-check-input {
   margin-right: var(--sgds-form-input-margin-right);
 }
+.form-check-label-container{
```

Comment:
seems like without this the label and feedback works well too since no gap in between